src/trajectory/Interpolated.cpp

```diff
@@ -100,6 +100,18 @@ void Interpolated::evaluateDerivative(double _t, int _derivative,
     _tangentVector.setZero();
   }
 
+  //if(_t==getStartTime())
+  if(_t==mWaypoints[0].t)
```

Comment:
This is comparing two floating point numbers for exact equality, which is almost always a bug. What is the purpose of thsi check?